src/pages/commits/ActiveCommits/ChartToggle.tsx

```diff
@@ -12,9 +13,14 @@ export const ChartToggle: React.FC<{
   currentChartType: ChartTypes;
   onChangeChartType: (chartType: ChartTypes) => void;
 }> = ({ currentChartType, onChangeChartType }) => {
+  const { sendEvent } = useProjectHealthAnalytics();
   const onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     const chartType = e.target.value as ChartTypes;
     onChangeChartType(chartType);
+    sendEvent({
```

Comment:
Noticed this error when toggling the chart toggle. I'm not sure if it was introduced in this pr or not. 
![image](https://user-images.githubusercontent.com/4605522/161628906-3e16cc38-1622-4b98-8092-ec42358754d7.png)

Reply:
https://jira.mongodb.org/browse/EVG-16689
Opened a ticket.